.github/workflows/security.yaml

```diff
@@ -0,0 +1,103 @@
+name: "Security Scanning"
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [main]
```

Comment:
We still want to run it on this right? But not make it required?

Reply:
Yeah that probably makes sense in case the PR introduces new security vulnerabilities otherwise they would not get caught until the next cron.